providers/standard/provider.yaml

```
@@ -93,3 +93,23 @@ config:
         type: string
         example: uv
         default: auto
+
+task-decorators:
+  - class-name: airflow.providers.standard.decorators.python.python_task
+    name: python
```

Ooohh, I wonder if we should do this:

```suggestion
  - class-name: airflow.providers.standard.decorators.python.python_task
    name: __call__
  - class-name: airflow.providers.standard.decorators.python.python_task
    name: python
```

That way we don't need to special case anything in the main decorator?

---

Good idea, let me try.